app/views/organizations/organization_members/index.html.slim

```diff
@@ -0,0 +1,2 @@
+h1 Organizations::OrganizationMembers#index
```

Comment:
Destroy, index and update files seem pretty default?

Reply:
whops pr should have been a draft